source/features/list-prs-for-file.tsx

```diff
@@ -102,7 +102,7 @@ const getPrsByFile = cache.function(async (): Promise<Record<string, number[]>>
 
 async function init(): Promise<void> {
 	// `clipboard-copy` on blob page, `#blob-edit-path` on edit page
-	const path = select('clipboard-copy, #blob-edit-path')!.getAttribute('value')!;
+	const path = (await elementReady('clipboard-copy, #blob-edit-path', {waitForChildren: false}))!.getAttribute('value')!;
```

Comment:
Check the elements it selects: If they don't have much content inside, `waitForChildren: false` doesn't make any difference.

Reply:
```suggestion
	const path = (await elementReady('clipboard-copy, #blob-edit-path'))!.getAttribute('value')!;
```